alts/worker/tasks.py

```diff
@@ -175,29 +181,52 @@ def set_artifacts_when_stage_has_unexpected_exception(
     module_name = task_params.get('module_name')
     module_stream = task_params.get('module_stream')
     module_version = task_params.get('module_version')
+
+    def get_excluded_packages():
+        uri = 'https://git.almalinux.org/almalinux/alts-exclusions/raw/branch/main/skipped_tests.json'
+        try:
+            response = requests.get(uri)
+            response.raise_for_status()
+            return response.json()
+        except requests.RequestException:
+            return {}
+
     try:
         # Wait a bit to not spawn all environments at once when
         # a lot of tasks are coming to the machine
         time.sleep(random.randint(5, 10))
-        runner.setup()
-        runner.run_system_info_commands()
-        runner.install_package(
-            package_name,
-            package_version=package_version,
-            package_epoch=package_epoch,
-            module_name=module_name,
-            module_stream=module_stream,
-            module_version=module_version,
-            semi_verbose=True,
-        )
-        if CONFIG.enable_integrity_tests:
-            runner.run_package_integrity_tests(package_name, package_version)
-        runner.run_third_party_tests(
-            package_name,
-            package_version=package_version,
-            package_epoch=package_epoch,
-        )
-        runner.uninstall_package(package_name)
+
+        test_steps = [
+            ("setup", runner.setup, [], {}),
+            ("run_system_info_commands", runner.run_system_info_commands, [], {}),
+            ("install_package", runner.install_package, [
+                package_name
+            ], {
+                 "package_version": package_version,
+                 "package_epoch": package_epoch,
+                 "module_name": module_name,
+                 "module_stream": module_stream,
+                 "module_version": module_version,
+                 "semi_verbose": True,
+             }),
+            ("run_package_integrity_tests", runner.run_package_integrity_tests, [
+                package_name, package_version
+            ], {}),
+            ("run_third_party_tests", runner.run_third_party_tests, [
+                package_name
+            ], {
+                 "package_version": package_version,
+                 "package_epoch": package_epoch,
+             }),
+            ("uninstall_package", runner.uninstall_package, [package_name], {})
+        ]
+        packages_to_skip = get_excluded_packages()
```

Comment:
I think it's better to move getting that mapping to the alts-scheduler service, with that we will be able to:
- implement a cache of this mapping (using `cachetools.TTLCache` as we do in the [build-node](https://github.com/AlmaLinux/albs-node/blob/master/build_node/build_node_supervisor.py#L28)) to reduce amount of requests to our gitea instance
- check on the scheduler side that integrity tests should be skipped or not and pass that value to the celery workers, something like `test_params.get('skip_integrity_tests', False)`